checks/model/members.js

```diff
@@ -0,0 +1,69 @@
+import '../../server/env';
+
+import logger from '../../server/lib/logger';
+import { sequelize } from '../../server/models';
+
+async function checkDeletedMembers({ fix = false } = {}) {
+  const message = 'No non-deleted Members without a matching non-deleted Collective (no auto fix)';
```

Comment:
Looks like there is an autofix

```suggestion
  const message = 'No non-deleted Members without a matching non-deleted Collective)';
```